src/browser/api_protocol/api_handlers/application.js

```diff
@@ -383,6 +384,67 @@ function runApplication(identity, message, ack, nack) {
     }
 }
 
+async function runApplications(identity, message, ack, nack) {
+    const { payload } = message;
+    const { applications } = payload;
+
+    const appsToRunWithRVM = [];
+
+    async function runManifest(application) {
+        const { uuid, manifestUrl } = application;
+
+        const appIdentity = apiProtocolBase.getTargetApplicationIdentity(application);
+        let remoteSubscriptionUnSubscribe;
+        const logMsg = manifestUrl ?
+            `unsub called before duplicate uuid transport listener removed for ${uuid}` :
+            '';
```

Comment:
maybe early return if there isn't a manifest? so this doesn't have to be a ternary?

Reply:
Early return added.